www/js/theme_editor.js

```diff
@@ -298,6 +298,11 @@ module.exports = {
   init() {
     const themeOptions = document.querySelector('#custom-theme-options');
 
+    const currentTheme = themes.find(theme => theme.key === store.getUserPref('app.theme'));
```

Comment:
Does this work? I think the store value needs to set before the render process is started (in `app.js`)